website/next.config.js

```diff
@@ -11,6 +11,11 @@ module.exports = withNextra({
     legacyBrowsers: false,
     images: { allowFutureImage: true },
   },
+  env: {
```

Comment:
This allows us to reference the PR branch so we don't have broken links for PRs if we add/rename files